### PR TITLE
222631: Upgraded to .Net 6.0

### PR DESCRIPTION
**Pull request by rajeshkaremane**
# Description
*Upgraded .Net framework version and dependencies*
* Upgraded docker image version to .Net 6.0*
* Migrated from Newtonsoft.Json to System.Text.Json*
*  Added swagger for dev environment *

*Link to the relevant work items:  [AB#222631](https://dev.azure.com/defragovuk/93c65135-d16b-49d6-a191-4a5313532779/_workitems/edit/222631)*

# Testing
*Build completed successfully locally*

### CODE REVIEW

**Review by jeevankuduvaravindran — APPROVED** [review]

File: helm/ffc-demo-payment-service-core/values.yaml [hunk 1]
```diff
@@ -53,17 +53,17 @@ readinessProbe:
...
   failureThreshold: 3
-  timeoutSeconds: 30
+  timeoutSeconds: 60
 
 livenessProbe:
   path: /healthz
   port: 3007
   initialDelaySeconds: 70
-  periodSeconds: 20
+  periodSeconds: 60
```
Comment (DaRoszko) [inline helm/ffc-demo-payment-service-core/values.yaml:64]: why 60 secs?

Comment (rajeshkaremane) [reply helm/ffc-demo-payment-service-core/values.yaml:64]: It was increased during debugging. Changing it back to original value of 20seconds

File: FFCDemoPaymentService/FFCDemoPaymentService.csproj [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
-    <Version>5.0.12</Version>
+    <Version>6.0.0</Version>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.3.0"/>
-    <PackageReference Include="Azure.Messaging.ServiceBus" Version="7.1.2"/>
-    <PackageReference Include="Microsoft.ApplicationInsights.AspNetCore" Version="2.17.0"/>
-    <PackageReference Include="Microsoft.AspNetCore.Diagnostics.HealthChecks" Version="2.2.0"/>
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Tools" Version="5.0.5">
+    <PackageReference Include="Azure.Identity" Version="1.10.0" />
```
Comment (DaRoszko) [inline FFCDemoPaymentService/FFCDemoPaymentService.csproj:8]: 1.10.2 ?

Comment (rajeshkaremane) [reply FFCDemoPaymentService/FFCDemoPaymentService.csproj:8]: Looks like version updated recently. will take it

File: FFCDemoPaymentService/FFCDemoPaymentService.csproj [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
-    <PackageReference Include="Microsoft.Extensions.Diagnostics.HealthChecks.EntityFrameworkCore" Version="5.0.5"/>
-    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="5.0.5.1"/>
-    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL.Design" Version="1.1.1"/>
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.1"/>
-    <PackageReference Include="SonarAnalyzer.CSharp" Version="8.21.0.30542">
+    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="7.0.4" />
+    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL.Design" Version="1.1.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
```
Comment (kbabigumira) [inline FFCDemoPaymentService/FFCDemoPaymentService.csproj:18]: should be using System.Text.Json
https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/migrate-from-newtonsoft?pivots=dotnet-6-0

File: FFCDemoPaymentService/Properties/launchSettings.json [hunk 4]
```diff
@@ -12,15 +12,15 @@
     "IIS Express": {
       "commandName": "IISExpress",
       "launchBrowser": true,
-      "launchUrl": "weatherforecast",
+      "launchUrl": "healthz",
```
Comment (kbabigumira) [inline FFCDemoPaymentService/Properties/launchSettings.json:15]: Why was this changed?

File: helm/ffc-demo-payment-service-core/Chart.yaml [hunk 5]
```diff
@@ -5,5 +5,5 @@ name: ffc-demo-payment-service-core
 version: 1.0.0
 dependencies:
 - name: ffc-helm-library
```
Comment (kbabigumira) [inline helm/ffc-demo-payment-service-core/Chart.yaml:7]: Wasn't this referencing the adp-helm-library?

File: FFCDemoPaymentService/Properties/launchSettings.json (hunk 4 above)
Comment (rajeshkaremane) [reply FFCDemoPaymentService/Properties/launchSettings.json:15]: because there is no endpoint weatherforecast in the dotnet code. It was a bug in the original code

File: FFCDemoPaymentService/FFCDemoPaymentService.csproj (hunk 3 above)
Comment (rajeshkaremane) [reply FFCDemoPaymentService/FFCDemoPaymentService.csproj:18]: migrated to System.Text.Json

**Review by kbabigumira — APPROVED** [review]

**Review by kbabigumira — APPROVED** [review]
